awx/main/models/jobs.py

```diff
@@ -166,6 +166,19 @@ class Meta:
             "facts at the end of a playbook run to the database and caching facts for use by Ansible."
         ),
     )
+    playbook_integrity_verified = models.BooleanField(
+        blank=True,
+        default=None,
+        null=True,
+        editable=False,
+        help_text=_('Overall result of playbook integrity verification'),
```

Comment:
That could make sense on a job, but I don't see how it makes sense on a template.

Reply:
Thank you very much for your review @AlanCoding .
I agree with that. Integrity check result should be stored in a job, and jobtemplate should not.
I will update this part.

Reply:
I have moved the fields from JobOption to Job model, and also the migration code has been re-generated.
Thank you.